portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -46,6 +46,7 @@
     private final Gson gson = new Gson();
     private final UserService credentials = UserServiceFactory.getUserService();
     private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    // How to try/catch JavaIOException as a private final variable
     private final LanguageServiceClient languageService = LanguageServiceClient.create();
```

Comment:
make it non-final and initialize it in `init()`.

Reply:
Ok, I still had to resort to using try and catch statements but it's working now - thanks